app/models/service_template_ansible_playbook.rb

```diff
@@ -208,6 +210,15 @@ def self.new_job_template_required?(info, action, service_template)
   end
   private_class_method :new_job_template_required?
 
+  def rollback_job_templates
+    auth_user = User.current_userid || 'system'
+    config_info = options[:config_info]
+    config_info.keys.each do |action|
+      ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScript
+        .delete_in_provider_queue(options[:manager_id], { :name => self.class.send(:build_name, name, action) }, auth_user)
```

Comment:
That does not seem like it is required since we are saying this would only happen during the create of the service template.